nncf/onnx/hardware/fused_patterns.py

```diff
@@ -11,70 +11,326 @@
+
+    pattern.add_edge(input_node, add_node)
+    pattern.add_edge(add_node, multiply_node)
+    return pattern
+
 
-from nncf.onnx.hardware.patterns import create_swish_activation
-from nncf.onnx.hardware.patterns import create_input_preprocessing_pattern
-from nncf.onnx.hardware.patterns import create_decomposed_batch_norm
-from nncf.onnx.hardware.patterns import create_scale_shift
-from nncf.onnx.hardware.patterns import create_bn_scale_shift_activation
-from nncf.onnx.hardware.patterns import create_linear_bn_scale_shift_activation
-
-
-def _get_onnx_hw_fused_patterns() -> HWFusedPatterns:
-    hw_fused_patterns = HWFusedPatterns()
-
-    linear_ops = GraphPattern()
-    linear_ops.add_node(**LINEAR_OPERATIONS)
-    hw_fused_patterns.register(linear_ops, LINEAR_OPERATIONS['label'], match=False)
-
-    batch_norm = GraphPattern()
-    batch_norm.add_node(**BATCH_NORMALIZATION_OPERATIONS)
-    decomposed_batch_norm = create_decomposed_batch_norm()
-    batch_norms = batch_norm | decomposed_batch_norm
-    hw_fused_patterns.register(batch_norms, BATCH_NORMALIZATION_OPERATIONS['label'], match=False)
-
-    atomic_activations = GraphPattern()
-    atomic_activations.add_node(**ATOMIC_ACTIVATIONS_OPERATIONS)
-    swish = create_swish_activation()
-    activations = atomic_activations | swish
-    hw_fused_patterns.register(activations, 'ACTIVATIONS', match=False)
-
-    arithmetic_ops = GraphPattern()
-    arithmetic_ops.add_node(**ARITHMETIC_OPERATIONS)
-    hw_fused_patterns.register(arithmetic_ops, ARITHMETIC_OPERATIONS['label'], match=False)
-
-    batch_norm_activations_permutation = batch_norms + activations | \
-                                         activations + batch_norms | \
-                                         batch_norms | activations
-
-    hw_fused_patterns.register(linear_ops + batch_norm_activations_permutation, 'LINEAR + BN_ACT_PERM',
-                               match=True)
-
-    hw_fused_patterns.register(create_linear_bn_scale_shift_activation(),
-                               'LINEAR + BN + SCALE_SHIFT + ACTIVATIONS', match=True)
-    hw_fused_patterns.register(create_bn_scale_shift_activation(),
-                               'BN + SCALE_SHIFT + ACTIVATIONS', match=True)
-    hw_fused_patterns.register(linear_ops + arithmetic_ops, 'LINEAR + ARITHMETIC', match=True)
-    hw_fused_patterns.register(batch_norms + activations, 'BN + ACTIVATIONS', match=True)
-    hw_fused_patterns.register(activations + batch_norms, 'ACTIVATIONS + BN', match=True)
-    hw_fused_patterns.register(arithmetic_ops + batch_norm_activations_permutation,
-                               'ARITHMETIC + BN_ACT_PERM', match=True)
-
-    input_preprocessing_pattern = create_input_preprocessing_pattern()
-    hw_fused_patterns.register(input_preprocessing_pattern,
-                               'INPUT_PREPROCESSING', match=True)
+@ONNX_HW_FUSED_PATTERNS.register(PatternNames.INPUT_MULTIPLY_SUBTRACT)
+def create_input_multiply_subtract():
```

Comment:
```suggestion
def create_input_scale_shift():
```

Reply:
Dropped this pattern & updated the original `create_input_scale_shift` with the `subtract` layer.